src/crypto/crypto_openssl.c

```diff
@@ -1218,6 +1218,43 @@ int crypto_get_random(void *buf, size_t len)
 int omac1_aes_vector(const u8 *key, size_t key_len, size_t num_elem,
 		     const u8 *addr[], const size_t *len, u8 *mac)
 {
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L
++	EVP_MAC_CTX *ctx = NULL;
++	EVP_MAC *emac;
```

Comment:
There are some typos, i.e., additional plus symbols should be removed.

Reply:
Ooo. Thanks. My mistake. Fixed.